### ISSUE TITLE
PowerShell script output encoding is messed up on linux

### ISSUE THREAD

**Issue by ed-ilyin** [PowerShell/PowerShell#2339]
## Steps to reproduce

1) on the ubuntu 16.04 put the following string to the test.ps1 file

``` powershell
@{aaa=111;bbb=$false;ccc='абба'} | ConvertTo-Json
```

2) run following command from terminal (urxvt + zsh in my case):

``` zsh
powershell ./test.ps1
```
## Expected behavior

Output should show "абба"

``` zsh
{
    "bbb":  false,
    "aaa":  111,
    "ccc":  "абба"
}
```
## Actual behavior

``` powershell
{
    "ccc":  "Ð°Ð±Ð±Ð°",
    "bbb":  false,
    "aaa":  111
}
```
## Environment data

``` powershell
> $PSVersionTable
Name                           Value                                                                      
----                           -----                                                                      
PSVersion                      6.0.0-alpha                                                                
PSEdition                      Core                                                                       
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                    
BuildVersion                   3.0.0.0                                                                    
GitCommitId                    v6.0.0-alpha.10                                                            
CLRVersion                                                                                                
WSManStackVersion              3.0                                                                        
PSRemotingProtocolVersion      2.3                                                                        
SerializationVersion           1.1.0.1
```

---

BTW: If in terminal I run following command, I get correct output:

``` powershell
 powershell '@{aaa=111;bbb=$false;ccc="абба"} | ConvertTo-Json'                                                             
{
    "bbb":  false,
    "aaa":  111,
    "ccc":  "абба"
}
```

**Comment by juergenhoetzel** [comment]
I can confirm.

A simpler example to reproduce the issue:

`write-host ä`

`powershell ./test.ps1` results in:
`Ã¤`

Seems like somehow the _Powershell_  trys to convert the output from _latin1_  to _UTF8_, i.e. assumes the script is in _latin1_ encoding. Because converting back (via external iconv)  fixes the output again:

```
powershell  ./test.ps1|iconv -f utf8 -t latin1
ä

```

**Comment by juergenhoetzel** [comment]
Update: Adding an  UTF-8 [Byte Order Mark](https://en.wikipedia.org/wiki/Byte_order_mark) to the script:

`sed -i '1s/^/\xef\xbb\xbf/' test.ps1`

fixes the issue. I wonder why _Powershell_ expects _latin1_ by default.

**Comment by vors** [comment]
Thanks @juergenhoetzel for the investigation. I think it's closely related to #707

**Comment by juergenhoetzel** [comment]
> Thanks @juergenhoetzel for the investigation. I think it's closely related to #707

Yes, both tasks are about the fact that [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) headers are not commonly used on GNU/Linux systems. 

This issue is about a better default encoding for reading PS1 files when the  [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) header is missing (latin1 is assumed by default). IMO a better default would be using the *locale* settings:

```
[juergen@pidsley test]$ localectl
   System Locale: LANG=de_DE.UTF-8
 

```

Whereas #707 is about using a better default for Output encoding, because Unix tools like *cat* doesn't utilize the BOM header and just display garbage.

**Comment by joeyaiello** [comment]
This has been fixed in `6.0.0-beta.1`.
